src/stubber/variants.py

```diff
@@ -0,0 +1,103 @@
+"""
+POC to create a variants of createstubs.py
+"""
+
+import subprocess
+import tempfile
+from typing import List, Union
+import libcst as cst
+import libcst.codemod as codemod
+from pathlib import Path
+
+from stubber.codemod.board import CreateStubsCodemod, CreateStubsVariant
+from stubber.codemod.modify_list import ListChangeSet
+
+from loguru import logger as log
+
+from stubber.utils.post import run_black
+from stubber.minify import minify, cross_compile
+
+ALL_VARIANTS = [CreateStubsVariant.BASE, CreateStubsVariant.MEM, CreateStubsVariant.DB, CreateStubsVariant.LVGL]
```

Comment:
```suggestion
ALL_VARIANTS = list(CreateStubsVariant)
```

Bit easier to maintain :)

Reply:
nice , did not know that syntax. does that work just because its an enum ?

Reply:
:+1: Yup. See https://github.com/python/cpython/blob/7a0dc8a802814503eb93662a3486b890de070734/Lib/enum.py#L797-L801